sunpy/util/decorators.py

```diff
@@ -182,3 +183,27 @@ def deprecate(obj, message=message, name=name, alternative=alternative):
         return deprecate(message)
 
     return deprecate
+
+
+class Appender(object):
+    """
+    A function decorator that will append and/or prepend an addendum
+    to the docstring of the target function.
+
+    If a dictionary(d) is provided as parameter then format by .format().
+    """
+    def __init__(self, addendum, append=True, prepend=False, d={}):
```

Comment:
Rather than having `append` and `prepend` as boolean arguments, why not have them as strings? Then you could prepend one string and append another?